client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```
@@ -30,7 +30,7 @@
 
 .import-focused .step-container__content,
 .site-setup.processing .step-container__content,
-.anchor-fm.processing .step-container__content {
+.step-container__content {
```

This feels like it will incorrectly make all step container content have 100% width. I think we should remove the rule entirely.
```suggestion
{
```